protocol.html

```diff
@@ -768,6 +699,44 @@ <h2 property="schema:name">Extensions</h2>
                 <h3 property="schema:name">Subscription Types Extension</h3>
                 <div datatype="rdf:HTML" property="schema:description">
                   <p><em>This section is non-normative.</em></p>
+
+                  <p>
+                    New subscription types may be defined and published independently of this document.
+                    This section describes some guidelines and recommendations for publishing new subscription types.
+                  </p>
+
+                  <p>
+                    Each new subscription type will declare a unique IRI for use with
+                    both discovery and with the establishment of new subscription connections.
+                  </p>
+
+                  <p>
+                    When defining IRIs for use with a subscription type, it is considered
+                    best practice to publish vocabulary and JSON-LD context documents
+                    in public locations.
```

Comment:
I have no strong preference but FWIW:

```suggestion
                    best practice to publish vocabulary and JSON-LD context documents
                    that are publicly accessible.
```